gensim/models/word2vec.py

```diff
@@ -606,7 +606,7 @@ def load_word2vec_format(cls, fname, fvocab=None, binary=False, norm_only=True):
                     result.syn0[line_no] = fromstring(fin.read(binary_len), dtype=REAL)
             else:
                 for line_no, line in enumerate(fin):
-                    parts = utils.to_unicode(line).split()
+                    parts = utils.to_unicode(line.rstrip()).split(" ")
```

Comment:
I have not tried in Python 3, and I am no Python expert. The problem is not with the new line. It is with the more exotic spaces, such as non-breaking space(formfeed?). If you stick to the same set of separation characters as in word2vec.c -> (if ((ch == ' ') || (ch == '\t') || (ch == '\n')) ) in some way, the problem can be fixed.